spec/features/articles_spec.rb

```diff
@@ -34,7 +34,6 @@
     end
 
     it "Lets you delete an article on the index page" do
-      
```

Comment:
the more code you write, the more you care about ensuring consistent whitespace. 👍